pkg/config/configobserver/configobserver.go

```diff
@@ -294,12 +319,28 @@ func (c *Controller) setConfigLocked(config *config.Controller) {
 			}
 		}
 	} else {
-		klog.V(2).Infof(
-			"Configuration set: enabled=%t endpoint=%s interval=%s username=%t token=%t reportEndpoint=%s initialPollingDelay=%s minRetryTime=%s pollingTimeout=%s",
-			config.Report, config.Endpoint, config.Interval, len(config.Username) > 0, len(config.Token) > 0, config.ReportEndpoint,
-			config.ReportPullingDelay, config.ReportMinRetryTime, config.ReportPullingTimeout)
+		klog.V(2).Infof( //nolint: gomnd
+			"Configuration updated: "+
```

Comment:
`set` became `updated`, maybe it's better to refactor it somehow. What about creating a function `toStr` on config?